python/tvm/meta_schedule/testing/torchbench/run.py

```diff
@@ -0,0 +1,301 @@
+        repeat time is lowered than this.
+        """,
+    )
+    args.add_argument(
+        "--disable-adaptive-training",
+        action="store_true",
+        help="Whether to disable adpative training for cost model.",
+    )
+    args.add_argument(
+        "--disable-cpu-flush",
+        action="store_true",
+        help="Whether to disable CPU cache flush.",
+    )
+
+    # RPC-related args
+    args.add_argument(
+        "--rpc-host",
+        type=str,
+        help="Host of the RPC Tracker for tuning. Use LocalRunner if not provided",
+    )
+    args.add_argument(
+        "--rpc-port",
+        type=int,
+        help="Port of the RPC Tracker for tuning",
+    )
+    args.add_argument(
+        "--rpc-key",
+        type=str,
+        help="Key of the RPC Tracker for tuning",
+    )
+
+    parsed = args.parse_args()
+    return parsed
+
+
+logging.basicConfig(
+    format="%(asctime)s.%(msecs)03d %(levelname)s %(message)s", datefmt="%Y-%m-%d %H:%M:%S"
+)
+logging.getLogger("tvm.meta_schedule").setLevel(logging.INFO)
+ARGS = parse_args()
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def get_metaschedule_runner():
+    if ARGS.rpc_host is not None:
+        return ms.runner.RPCRunner(
+            rpc_config=ms.runner.RPCConfig(
+                tracker_host=ARGS.rpc_host,
+                tracker_port=ARGS.rpc_port,
+                tracker_key=ARGS.rpc_key,
+                session_timeout_sec=600,
+            ),
+            evaluator_config=ms.runner.EvaluatorConfig(
+                number=ARGS.number,
+                repeat=ARGS.repeat,
+                min_repeat_ms=ARGS.min_repeat_ms,
+                enable_cpu_cache_flush=ARGS.cpu_flush,
+            ),
+            alloc_repeat=1,
+        )
+    else:
+        return ms.runner.LocalRunner()
+
+
+def create_tvm_backend():
+    def backend(graph_module, example_inputs):
+        jit_mod = torch.jit.trace(graph_module, example_inputs)
+        shape_list = [(f"inp_{idx}", i.shape) for idx, i in enumerate(example_inputs)]
+        ir_mod, params = tvm.relay.frontend.from_pytorch(jit_mod, shape_list)
+
+        lib = ms.tune_relay(
+            mod=ir_mod,
+            target=ARGS.target,
+            config=ms.TuneConfig(
+                strategy="evolutionary",
+                num_trials_per_iter=64,
+                max_trials_per_task=ARGS.num_trials,
+                max_trials_global=ARGS.num_trials,
+                adaptive_training=not ARGS.disable_adaptive_training,
+            ),
+            runner=get_metaschedule_runner(),
+            work_dir=ARGS.work_dir,
+            params=params,
+            backend=ARGS.backend,
+        )
+
+        device = tvm.cuda(0) if ARGS.target.kind.name == "cuda" else tvm.cpu(0)
+
+        if ARGS.backend == "graph":
+            mod = graph_executor.GraphModule(lib["default"](device))
+        elif ARGS.backend == "vm":
+            raise RuntimeError("vm backend not supported yet")
```

Comment:
You may refer to https://github.com/apache/tvm/blob/e8aeb4adf3525837db5f24965104640163b38f0e/python/tvm/meta_schedule/testing/tune_utils.py#L107 for vm usage.